docker/Dockerfile

```diff
@@ -13,10 +13,7 @@ RUN rm -f /etc/apt/apt.conf.d/docker-clean \
   && echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' >/etc/apt/apt.conf.d/keep-cache
 
 # Install necessary packages
-RUN \
-  --mount=type=cache,target=/var/cache/apt,sharing=locked \
-  --mount=type=cache,target=/var/lib/apt,sharing=locked \
-  apt-get update \
+RUN apt-get update \
```

Comment:
why do you remove the build cache?

This question of course also applys to all further removements of the build cache, but not adding the same comment now x times 😅

Reply:
See the notes above and in [f70fb02](https://github.com/invoke-ai/InvokeAI/pull/2722/commits/f70fb0278c8d460bb5ad1f51e3cb536560ffe727) it breaks podman unfortunately.